src/index.html

```diff
@@ -8,7 +8,7 @@
       href="./styles/main.scss"
```

Comment:
The stylesheet link references a `.scss` file. Browsers cannot directly interpret SCSS files; they need to be compiled into CSS. Ensure that the SCSS file is compiled to a CSS file, and update the `href` attribute to point to the resulting CSS file.